config/settings/production.py

```diff
@@ -205,6 +205,10 @@
     os.getenv("ROSETTA_CLIENT_VERIFY_CERTIFICATES", "True")
 )
 
+# Feature flag
+# Process record detail apis sequentially or in parallel
+ENABLE_PARALLEL_API_CALLS: bool = True
```

Comment:
Would it make sense to source the ENABLE_PARALLEL_API_CALLS value from the environment, allowing it to be changed in Prod to evaluate both options?